modules/allocation-manager/airavata-allocation-manager/airavata-allocation-manager-server/src/main/java/org/apache/airavata/allocation/manager/db/entities/UserAllocationDetailEntity.java

```diff
@@ -0,0 +1,162 @@
+package org.apache.airavata.allocation.manager.db.entities;
+
+import java.io.Serializable;
+import javax.persistence.*;
+import java.math.BigInteger;
+
+
+/**
+ * The persistent class for the USER_ALLOCATION_DETAILS database table.
+ * 
+ */
+@Entity
+@Table(name="USER_ALLOCATION_DETAILS")
+@NamedQuery(name="UserAllocationDetailEntity.findAll", query="SELECT u FROM UserAllocationDetailEntity u")
+public class UserAllocationDetailEntity implements Serializable {
+	private static final long serialVersionUID = 1L;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name="PROJECT_ID")
+	private int projectId;
+
+	@Column(name="ALLOCATION_STATUS")
+	private String allocationStatus;
+
+	private String comments;
+
+	@Column(name="DISK_USAGE_RANGE_PER_JOB")
+	private BigInteger diskUsageRangePerJob;
+
+	@Lob
+	private byte[] documents;

```

Comment:
It is better to move Lob entries to a new table like Documents and keeping the relationship to that table. This will improve the search performance of this table